src/components/App/App.css

```diff
@@ -19,7 +18,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  height: 90%;
+  height: 70vh;
```

Comment:
Making this view height helped with a bit of styling, I honestly can not recall where.